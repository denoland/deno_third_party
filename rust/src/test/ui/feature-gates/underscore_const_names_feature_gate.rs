const _: () = (); //~ ERROR is unstable

fn main() {}
