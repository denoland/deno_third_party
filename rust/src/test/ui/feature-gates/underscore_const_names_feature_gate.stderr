error[E0658]: naming constants with `_` is unstable (see issue #54912)
  --> $DIR/underscore_const_names_feature_gate.rs:1:1
   |
LL | const _: () = (); //~ ERROR is unstable
   | ^^^^^^^^^^^^^^^^^
   |
   = help: add #![feature(underscore_const_names)] to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
